js/foundation.util.mediaQuery.js

```diff
@@ -173,7 +178,7 @@ var MediaQuery = {
    * @private
    */
   _watcher() {
-    $(window).on('resize.zf.mediaquery', () => {
+    $(window).off('resize.zf.mediaquery').on('resize.zf.mediaquery', () => {
```

Comment:
BTW ... it will be great if you can explain this one above code to a js beginner like me :wink: